components/brex_staging/actions/list-transactions-for-selected-cash-account/list-transactions-for-selected-cash-acccount.mjs

```diff
@@ -0,0 +1,33 @@
+import brexApp from "../../brex_staging.app.mjs";
+import common from "../../../brex/actions/list-transactions-for-selected-cash-account/common.mjs";
+
+export default {
+  ...common,
+  name: "List Transactions for Selected Cash Account",
+  description: "Lists all transactions for the specified cash account. [See the docs here](https://developer.brex.com/openapi/transactions_api/#operation/listCashTransactions).",
```

Comment:
This filename has 3 c's in the word account - "acccount". It should be `list-transactions-for-selected-cash-account.mjs`